code/datums/martial/brassknuckles.dm

```diff
@@ -0,0 +1,30 @@
+/datum/martial_art/brassknuckles
+	name = "Brass Knuckles"
+	id = MARTIALART_BRASSKNUCKLES
+
+/datum/martial_art/brassknuckles/proc/punch(mob/living/A, mob/living/D)
+	var/obj/item/bodypart/affecting = D.get_bodypart(ran_zone(A.zone_selected))
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+	var/atk_verb = pick("pummels", "smokes", "wallops", "knucks", "knuckledusts")
+	D.visible_message("<span class='danger'>[A] [atk_verb]s [D]!</span>", \
+					"<span class='userdanger'>[A] [atk_verb]s you!</span>", null, null, A)
+	to_chat(A, "<span class='danger'>You [atk_verb] [D]!</span>")
+	playsound(get_turf(D), 'sound/weapons/punch1.ogg', 25, TRUE, -1)
+	log_combat(A, D, "knuckledusted (brass knuckles))")
+	D.apply_damage(20, BRUTE, affecting)
```

Comment:
jesus christ these knuckles do more damage than a spear to the face

Reply:
This makes sense but 20 damage makes this crit just as fast as a fire axe (24).

IMO you should lower this significantly and make it cause bone fractures easier than normal weapons (because that's what brass knuckles do).

Reply:
Yeah that's true. I don't know how to cause bone fractures so I just assumed high damage did it. Thank you for the suggestions!